app/policies/user_policy.rb

```diff
@@ -0,0 +1,38 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+    def show?
+      true
+    end
+
+    def feed?
```

Comment:
For the feed and discover policies, you should rethink the value being returned in this action to make sure that only the logged in user can visit the routes /current_user/feed or /current_user/discover.